src/webviews/extension-side/import-export/jupyterExporter.ts

```diff
@@ -8,7 +8,7 @@ import { Uri } from 'vscode';
 import { CellMatcher } from '../../../interactive-window/editor-integration/cellMatcher';
```

Comment:
Side note for myself (@rebornix). The IW component depends on `INotebookExporter` directly, maybe we should consider moving it into `src/notebooks`.